### PR TITLE
Statdeath experience fix

### PR DESCRIPTION
**Pull request by Marsunpaisti**

### CODE REVIEW

File: wurst/systems/core/Experience.wurst [hunk 1]
```diff
@@ -96,7 +96,7 @@ init
                 let pos = troll.getPos()
                 let killer = ENUM_GROUP
                     ..enumUnitsInRange(pos, expRange, Filter(() -> begin
-                        return GetFilterUnit().getOwner().getId() < bj_MAX_PLAYER_SLOTS
+                        return GetFilterUnit().getOwner() != players[PLAYER_NEUTRAL_AGGRESSIVE] and GetFilterUnit().getOwner() != players[PLAYER_NEUTRAL_PASSIVE]
```
Comment (theQuazz) [inline wurst/systems/core/Experience.wurst]: Let's just do `GetFilterUnit().isType(UNIT_TYPE_HERO)`

Comment (jonathanmclaus) [reply wurst/systems/core/Experience.wurst]: New meta to feed yourself to the One so it can kill the enemy fighting it.

Comment (jonathanmclaus) [reply wurst/systems/core/Experience.wurst]: `GetFilterUnit().getOwner().getTroll() != null` would work, right?

Comment (theQuazz) [reply wurst/systems/core/Experience.wurst]: Probably

File: wurst/systems/core/Experience.wurst [hunk 2]
```diff
@@ -96,7 +96,7 @@ init
                 let pos = troll.getPos()
                 let killer = ENUM_GROUP
                     ..enumUnitsInRange(pos, expRange, Filter(() -> begin
-                        return GetFilterUnit().getOwner().getId() < bj_MAX_PLAYER_SLOTS
```
Comment (jonathanmclaus) [inline wurst/systems/core/Experience.wurst]: How did the previous logic match neutral units? Aren't `NEUTRAL_PASSIVE` and `NEUTRAL_HOSTILE` both at least `bj_MAX_PLAYER_SLOTS`?

Comment (Marsunpaisti) [reply wurst/systems/core/Experience.wurst]: bj_MAX_PLAYER_SLOTS returned 28 in my tests, NEUTRAL_AGGRESSIVE is index 27 I believe.

Comment (jonathanmclaus) [reply wurst/systems/core/Experience.wurst]: Ah, thought it would return 24.

**Pull request merged**